yt/yql/providers/yt/provider/yql_yt_datasink.cpp

```diff
@@ -258,9 +262,24 @@ class TYtDataSink : public TDataProviderBase {
             TExprNode::TListType children = node->ChildrenList();
             children.resize(3);
             return ctx.NewCallable(node->Pos(), TYtDropTable::CallableName(), std::move(children));
+        } else if (mode && *mode == EYtWriteMode::Create) {
+            if (!node->Child(3U)->IsCallable("Void")) {
+                ctx.AddError(TIssue(ctx.GetPosition(node->Child(3U)->Pos()), TStringBuilder()
+                    << "Expected Void, but got: " << node->Child(3U)->Content()));
+                return {};
+            }
+
+            auto children = node->ChildrenList();
+            children.resize(6U);
+            const auto settings = node->Child(4U);
+            children[3U] = NYql::GetSetting(*settings, EYtSettingType::Columns)->TailPtr();
```

Comment:
What will happen if Write doesn't have Columns/OrderBy setting? We should probably issue error here